subsys/nrf_compress/src/lzma.c

```diff
@@ -104,15 +137,89 @@ static CLzma2Dec lzma_decoder;
 static CLzmaDec lzma_decoder;
 #endif
 
+#if CONFIG_NRF_COMPRESS_DICTIONARY_CACHE_SIZE > 0
+/**
+ * @brief Synchronize dictionary cache with external dictionary.
+ *
+ * This function synchronizes data in cache with external
+ * dictionary and proceeds with cache window.
+ *
+ * @param handle pointer to Lzma dictionary handle struct, for dictionary size reference.
+ *
+ * @retval 0 on successful synchronization
+ * @retval -EIO on any error with reading/writing to external dictionary
+ *
+ */
+static int synchronize_cache(const DictHandle *handle)
+{
+	SizeT const dict_write_size = cache.dict_pos_end - cache.dict_pos_begin + 1;
+
+	if (ext_dict->write(cache.dict_pos_begin,
+			cache.data, dict_write_size) != dict_write_size) {
```

Comment:
alignment issues here too. Also likely issue with line length, line length is 100 chars, this whole line is 101 characters, so why is splitting on the 50th character when it could split on the 83rd and be much more readable?